app/commands.py

```diff
@@ -1256,10 +1256,22 @@ async def server(ctx: Context) -> Optional[str]:
     # cmyui v1.7.3 | datadog v0.40.1 | geoip2 v4.1.0
     # maniera v1.0.0 | mysql-connector-python v8.0.23 | orjson v3.5.1
     # psutil v5.8.0 | py3rijndael v0.3.3 | uvloop v0.15.2
-    reqs = (Path.cwd() / "requirements.txt").read_text().splitlines()
+    requirements = []
+
+    for line in (Path.cwd() / "requirements.txt").read_text().splitlines():
+        line = line.split(";")[0].strip()
+
+        if line.startswith("git+"):
+            split = line.split("@")[0].split("/")[-2:]
+            requirements.append(f"git:{'/'.join(split)}")
```

Comment:
Expected behaviour for git packages is to provide the `#egg={pkgname}` info

Reply:
what does the egg mean? I'd like to show both the github user and the repository name, only the repository name won't get you far

Reply:
`egg` is the package name of the package when installed

Reply:
see: [pip requirements format](https://pip.pypa.io/en/stable/reference/requirements-file-format/)